core/contracts/FixedPoint.sol

```diff
@@ -28,11 +28,31 @@ library FixedPoint {
         return a.value > b.value;
     }
 
+    /** @dev Whether `a` is greater than `b`. */
+    function isGreaterThan(Unsigned memory a, uint b) internal pure returns (bool) {
```

Comment:
If you don't want to add tests for these in this PR, can we add a TODO to add them in a follow up?